coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -2496,3 +2496,13 @@ def std(context, node):
 def copy_(context, node):
     inputs = _get_inputs(context, node, expected=3)
     context.add(mb.identity(x=inputs[0], name=node.name))
+
+@register_torch_op
```

Comment:
Can you add corresponding tests to https://github.com/apple/coremltools/blob/master/coremltools/converters/mil/frontend/torch/test/test_torch_ops.py?